Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs

```diff
@@ -29,11 +31,10 @@ public override void OnInspectorGUI()
                 EditorGUILayout.HelpBox("glTF path was not discovered in the streaming assets folder. Please copy over files to test example scene", MessageType.Warning);
             }
 
-            if (GUILayout.Button("Copy GLTF Directory to Streaming Assets"))
+            if (GUILayout.Button("Copy models to StreamingAssets"))
             {
-                string modelPath = MixedRealityToolkitFiles.MapRelativeFolderPathToAbsolutePath(MixedRealityToolkitModuleType.Examples, GLTFModelsPath);
-                DirectoryCopy(modelPath, Path.Combine(Application.streamingAssetsPath, "GltfModels"));
-                AssetDatabase.Refresh(ImportAssetOptions.ForceUpdate);
```

Comment:
Should this line be restored? Feels necessary

Reply:
it's back